qa/tasks/vstart_runner.py

```diff
@@ -711,13 +717,14 @@ def get_keyring_path(self):
         # to avoid assumptions about daemons' pwd
         return os.path.abspath("./client.{0}.keyring".format(self.client_id))
 
-    def run_shell(self, args, wait=True, stdin=None, check_status=True, omit_sudo=True):
+    def run_shell(self, args, wait=True, stdin=None, stdout=None, stderr=None,
+                  check_status=True, omit_sudo=True):
         # FIXME maybe should add a pwd arg to teuthology.orchestra so that
```

Comment:
teuthology has already `cwd` for this.

Reply:
Yeah, right. And I have remove the relating code in run_shell() in the parent class.